src/libraries/Common/tests/System/Runtime/CompilerServices/RuntimeHelpers.cs

```diff
@@ -0,0 +1,45 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#if (NETSTANDARD2_0 || NETFRAMEWORK)
```

Comment:
Should we just make the include conditional in whatever csproj this is in?

Reply:
Yes, we can do that too.

Reply:
Applied the suggestion. Thanks!